show/bgp_frr_v4.py

```diff
@@ -102,10 +110,16 @@ def neighbors(ipaddress, info_type, namespace):
 def network(ipaddress, info_type, namespace):
     """Show IP (IPv4) BGP network"""
 
-    if multi_asic.is_multi_asic() and namespace not in multi_asic.get_namespace_list():
-        ctx = click.get_current_context()
-        ctx.fail('-n/--namespace option required. provide namespace from list {}'\
-            .format(multi_asic.get_namespace_list()))
+    namespace = namespace.strip()
+    if multi_asic.is_multi_asic():
+        if namespace == multi_asic.DEFAULT_NAMESPACE:
+            ctx = click.get_current_context()
+            ctx.fail('-n/--namespace option required. provide namespace from list {}'
+                    .format(multi_asic.get_namespace_list()))
```

Comment:
add 1 more space at beginning of line 118 and 122

Reply:
done